spec/models/manageiq/providers/ansible_tower/automation_manager/credential_mixin_spec.rb

```diff
@@ -0,0 +1,25 @@
+describe ManageIQ::Providers::AnsibleTower::AutomationManager::CredentialMixin do
+  let(:credential) { FactoryGirl.create(:authentication) }
+  let(:credential_class) do
```

Comment:
This was updated during review.  Pasting my original comment since it still applies...

Please don't create class constants in the test environment.  They are not limited in scope to this test and have conflicted in the past and troubleshooting is painful.  Instead, if needed:
```ruby
let(:test_class) do
  class.new(::Authentication) do
    ATTRIBUTES = []
    etc...
  end
end
```